primitives/signed.fil

```diff
@@ -8,17 +8,18 @@ extern "signed.sv" {
    ) where 'L > 'G, WIDTH > 0;
 }
 
-comp SignedConst[WIDTH, VALUE]<'G: 1>() -> (
-   out: ['G, 'G + 1] WIDTH,
+comp SignedConst[WIDTH, VALUE, ?D=1]<'G: D>() -> (
+   out: ['G, 'G+D] WIDTH,
```

Comment:
This component is another example of something that can have multiple events but doesn't need any compilation support for them.